themes/bootstrap5/scss/mixin-overrides/color-mode.scss

```diff
@@ -0,0 +1,25 @@
+/* Override of built-in color mode, to output both types */
+
+@mixin color-mode($mode: light, $root: false) {
```

Comment:
I am overriding the provided color-mode mixin, which supports only one mode at a time -- either system-defined or DOM-defined color-scheme, but not both.  So, I'm outputting the CSS for both types.